ingestion/src/metadata/ingestion/source/database/column_type_parser.py

```diff
@@ -192,6 +192,7 @@ class ColumnTypeParser:
         "UUID": "UUID",
         "POINT": "POINT",
         "POLYGON": "POLYGON",
+        "AggregateFunction()": "AGGREGATEFUNCTION",
```

Comment:
do we need brackets here?

Reply:
Yes, so we are getting it as `AggregateFunction()`.